docs/photoniq/vwrs/waitingroom.md

```diff
@@ -212,3 +223,25 @@ Below is a sample HTML template for a waiting room page. This page is displayed
   </body>
 </html>
 ```
+
+In above HTML `queue-status` API returns the following response:
+```
+  {
+    "avg_waiting_time" : 19871.8,
+    "backoff_interval" : 1,
+    "position" : 99359,
+    "queue_depth" : 0,
+    "rate_limit" : 5,
+    "waiting_room_interval" : 1
+  } 
+```
+
+- avg_waiting_time: The average time users spend in the virtual waiting room before being granted access to the origin service (in milliseconds).
+- backoff_interval: Indicates the interval in seconds after which the EdgeWorker should call the status API call to the vwrs server. This is used to reduce server load.
```

Comment:
```suggestion
- backoff_interval: The time interval (in seconds) for refreshing the queue position in the waiting room.
```

Reply:
Trying to keep this a little vague since it is used by us internally.

@naimishvora-macrometa wondering if we can accomplish this without exposing this.  For this PR, continue to document here, but maybe in the future we could make this value not public.  It just provides for questions to be asked and really the customer can't do much with this value.  It is just for our consumption.

Reply:
Yes we can let me remove backoff_interval